.github/workflows/win.yml

```diff
@@ -9,9 +9,10 @@ jobs:
     runs-on: windows-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+      - name: Setup java
+        uses: joschi/setup-jdk@v2
         with:
-          java-version: '16-ea' 
+          java-version: 15
```

Comment:
probably java-version: 16

Reply:
This will be updated to 16 once Java 16 is released i.e. March 16.